lib/ops/opsclient/opsclient.go

```diff
@@ -870,10 +870,15 @@ func (c *Client) RotateSecrets(req ops.RotateSecretsRequest) (*ops.RotatePackage
 }
 
 // RotatePlanetConfig rotates planet configuration package for the server specified in the request
-func (c *Client) RotatePlanetConfig(req ops.RotatePlanetConfigRequest) (*ops.RotatePackageResponse, error) {
+func (c *Client) RotatePlanetConfig(req ops.RotateConfigPackageRequest) (*ops.RotatePackageResponse, error) {
 	return nil, trace.NotImplemented("this method is only supported by local operator")
 }
 
+// RotateTeleportConfig rotates teleport configuration package for the server specified in the request
+func (c *Client) RotateTeleportConfig(req ops.RotateConfigPackageRequest) (*ops.RotatePackageResponse, *ops.RotatePackageResponse, error) {
```

Comment:
I imagine this would require a lot of refactoring e.g. because Operator we use in upgrade FSM has to call methods of both "local" and "not local" ops service.